src/tests/utils/convert.js

```diff
@@ -120,7 +120,7 @@ describe('convert render functions', () => {
       const wbDef = definitions.noAuth;
       return convert.renderIndex(wbDef)
         .then(string => {
-          string.should.containEql('authentication: {}');
+          string.should.not.containEql('authentication:');
```

Comment:
Originally, we generated an empty `{}` when a WB app doesn't have auth. The right way is not to include an `authentication` section at all if the WB app doesn't have auth.